api/routes.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+import (
+	"github.com/go-macaron/binding"
```

Comment:
nope.  This package is not versioned and doesnt use gopkg.in.   It does internally reference gokpg.in/macaron.v1 though